BeagleBone/AI/pru/AM335x_PRU.cmd

```diff
@@ -0,0 +1,86 @@
+/****************************************************************************/
```

Comment:
The filename isn’t accurate. Just remove the SoC name if needed.

Reply:
Found AM57xx_PRU.cmd in /usr/lib/ti/pru-software-support-package/examples/am572x/PRU_RPMsg_Echo_Interrupt1_0
There are a few differences....